api/transport/router.go

```diff
@@ -60,3 +67,22 @@ type RouteTable interface {
 	// Registers zero or more procedures with the route table.
 	Register([]Procedure)
 }
+
+// ProceduresByServiceProcedure offers to sort a slice of Procedure by Service
+// & Procedure.
+type ProceduresByServiceProcedure []Procedure
```

Comment:
I reused the existing naming + some changes that got intertwined during a rebase changing the naming to this as well. I am not against a better name!